alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/var/buildkite
+export GEM_HOME=${HOME}/.gem
+export BUILDKITE_AGENT_VERSION=edge
+export BUILDKITE_BUILD_PATH=/var/buildkite/builds
+export BUILDKITE_HOOKS_PATH=/var/buildkite/hooks
+export BUILDKITE_BOOTSTRAP_SCRIPT_PATH=/usr/lib/buildkite/bootstrap.sh
+
+exec chpst -u buildkite -- /usr/bin/buildkite-agent start
```

Comment:
Is the `buildkite-agent` binary not in your `$PATH`? if it is, then you could remove the `/usr/bin` prefix.